tests/pytests/functional/modules/test_cmdmod.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+import salt.modules.cmdmod as cmdmod
+import salt.utils.path
+
+pytestmark = [
+    pytest.mark.windows_whitelisted
+]
+
+
+@pytest.mark.skip_unless_on_windows(reason="Only run on Windows")
+def test_powershell():
+    """
+    Test cmd.powershell
+    """
+    ret = cmdmod.powershell("Write-Output foo")
+    assert ret == 'foo'
+
+
+@pytest.mark.skip_unless_on_windows(reason="Only run on Windows")
+def test_powershell_encode_cmd():
+    """
+    Test cmd.powershell with encode_cmd
+    """
+    ret = cmdmod.powershell('Write-Output "encoded foo"', encode_cmd=True)
+    assert ret == 'encoded foo'
+
+
+@pytest.mark.skip_unless_on_windows(reason="Only run on Windows")
+@pytest.mark.skipif(not salt.utils.path.which("pwsh"), reason="Powershell 7 Not Present")
```

Comment:
```suggestion
@pytest.mark.skip_if_binaries_missing("pwsh", message="Powershell 7 Not Present")
```

Reply:
Thanks... that's my ignorance in not knowing the names of all the skip things...